src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -105,6 +105,28 @@ object SnowflakeTelemetry {
     }
   }
 
+  private val MAX_CACHED_SPARK_PLAN_STATISTIC_COUNT = 1000
+
+  private[snowflake] def addSparkPlanStatistic(statistic: Set[String]): Unit = {
+    if (statistic.nonEmpty) {
+      this.synchronized {
+        if (logs.length >= MAX_CACHED_SPARK_PLAN_STATISTIC_COUNT) {
+          return
+        }
+      }
+
+      val metric: ObjectNode = mapper.createObjectNode()
+      val arrayNode = metric.putArray(TelemetryFieldNames.STATISTIC_INFO)
```

Comment:
why array?

Reply:
could you write a comment to show an example of message?

Reply:
There is a sample message in PR description.